src/app/Plans/components/VMMigrationDetails.tsx

```diff
@@ -0,0 +1,217 @@
+import * as React from 'react';
+import { useRouteMatch } from 'react-router-dom';
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  Button,
+  Card,
+  CardBody,
+  Pagination,
+  PageSection,
+  Title,
+} from '@patternfly/react-core';
+import {
+  Table,
+  TableHeader,
+  TableBody,
+  TableVariant,
+  sortable,
+  classNames as classNamesTransform,
+  ICell,
+  IRow,
+  wrappable,
+  expandable,
+} from '@patternfly/react-table';
+import { Link } from 'react-router-dom';
+import { useSelectionState } from '@konveyor/lib-ui';
+import spacing from '@patternfly/react-styles/css/utilities/Spacing/spacing';
+import alignment from '@patternfly/react-styles/css/utilities/Alignment/alignment';
+
+import { IVMStatus } from '@app/queries/types';
+import { useSortState, usePaginationState } from '@app/common/hooks';
+import PlanActionsDropdown from './PlanActionsDropdown';
+import VMStatusTable from './VMStatusTable';
+import { vmStatus1, vmStatus2 } from '@app/queries/mocks/plans.mock';
+import PipelineSummary from '@app/common/components/PipelineSummary';
+
+// TODO: Replace with final data definitions
+interface ITempMockMigration {
+  id: string;
+  schedule: {
+    begin: string;
+    end: string;
+  };
+  status: IVMStatus;
+  other: {
+    copied: number;
+    total: number;
+    status: string;
+  };
+}
+
+const mockMigration1: ITempMockMigration = {
+  id: 'VM1',
+  schedule: {
+    begin: '09 Aug 2019, 08:19:11',
+    end: '09 Aug 2019, 12:33:44',
+  },
+  status: vmStatus1,
+  other: {
+    copied: 93184,
+    total: 125952,
+    status: 'Ready',
+  },
+};
+
+const mockMigration2: ITempMockMigration = {
+  id: 'VM2',
+  schedule: {
+    begin: '09 Aug 2019, 08:19:11',
+    end: '09 Aug 2019, 09:43:12',
+  },
+  status: vmStatus2,
+  other: {
+    copied: 87952,
+    total: 87952,
+    status: 'Running',
+  },
+};
+
+const MOCK_TEMP_MIGRATIONS: ITempMockMigration[] = [mockMigration1, mockMigration2];
```

Comment:
Can we move these to their own mocks file in `@app/queries/mocks/migrations.mock.ts`, or maybe add them to `plans.types.ts`? Even if we're going to replace them later it would de-clutter this a bit.

Reply:
Yes, sorry about that, I consolidated the migration mock data.

Reply:
No need to apologize! 👍